lib/google-blog-feed-loader.js

```diff
@@ -0,0 +1,91 @@
+var FEED_URL = 'https://blog.webmaker.org/tag/teachtheweb/feed';
+var DEFAULT_CONTENT_SNIPPET_WORD_COUNT = 70;
+var SCRIPT_SRC = 'https://www.google.com/jsapi';
+
+// http://stackoverflow.com/a/822464
+var HTML_TAG_RE = /<(?:.|\n)*?>/gm;
+
+var scriptLoading = false;
+var scriptLoadCallbacks = [];
+
+function handleScriptLoad() {
```

Comment:
I was thinking of doing that, but it's not really necessary at all to make the code work, and just adds extra complexity. If anything, I guess I'd rather rename the variable to something a bit more accurate, like, um, `scriptTagInjectedIntoDOM` maybe? That would definitely make the variable less misleading.

I suppose an alternative would be to just call it `scriptEl`, initialize it to `null`, and have it be a reference to the actual `<script>` element, which we currently define as a function-level variable called `gScript` in `handleScriptLoad()`--then we can get rid of `gScript` and just test whether `scriptEl === null` instead of having the `scriptLoaded` boolean. Not a bad simplification, and makes the code more straightforward too!